### PR TITLE
fix/gh-1317-include-inactive-employees-option-for-teams

### PR DESCRIPTION
**Pull request by slobodanc93**
Resolves: #1317

### CODE REVIEW

File: src/main/java/eu/execom/hawaii/service/DataExportService.java [hunk 1]
```diff
@@ -109,38 +108,34 @@ private String getAbsenceName(AbsenceType absenceType) {
...
 
-  public List<Allowance> getAllowancesForExport(int year, Long teamId, boolean includeInactiveEmployees) {
-    List<UserStatusType> userStatusTypes = getUserStatusTypes(includeInactiveEmployees);
-    return allowanceRepository.findAllByYearTeamIdAndUserStatusType(year, teamId, userStatusTypes);
+  private String assignUserStatusTypeName(List<UserStatusType> userStatusTypes) {
+    List<String> absenceTypesToString = userStatusTypes.stream()
+                                                       .map(u -> u.toString().toLowerCase())
+                                                       .collect(Collectors.toList());
+
+    return "-" + String.join("-", absenceTypesToString);
```
Comment (stanislavgrujic) [inline src/main/java/eu/execom/hawaii/service/DataExportService.java:135]: You could use instead:
```
List<String> absenceTypesToString = userStatusTypes.stream()
                                                        .map(u -> u.toString().toLowerCase())
                                                        .collect(Collectors.joining("-"));
```

Comment (popovicbojan) [reply src/main/java/eu/execom/hawaii/service/DataExportService.java]: I fixed this. I noticed that merge is approved but the frontend was not reviewed, should we wait for the review?

File: src/main/frontend/src/pages/data-export/DataExport.js [hunk 2]
```diff
@@ -119,18 +121,53 @@ class DataExport extends Component {
...
+                        />
+                        {values.reportType === USERS_REPORT && (
+                          <InputPair
+                            view="switch"
+                            caption="Scheduled employees"
+                            name="includeScheduledEmployees"
+                            handleChange={checked => {
+                              setFieldValue(
+                                'includeScheduledEmployees',
+                                checked ? true : false
```
Comment (aberonja) [inline src/main/frontend/src/pages/data-export/DataExport.js:147]: Could this be just checked?

File: src/main/frontend/src/pages/data-export/DataExport.js [hunk 3]
```diff
@@ -119,18 +121,53 @@ class DataExport extends Component {
...
+                          />
+                        )}
+                        <InputPair
+                          view="switch"
+                          caption="Archived employees"
+                          name="includeArchivedEmployees"
+                          handleChange={checked => {
+                            setFieldValue(
+                              'includeArchivedEmployees',
+                              checked ? true : false
```
Comment (aberonja) [inline src/main/frontend/src/pages/data-export/DataExport.js:162]: Same as above.

File: src/main/frontend/src/pages/data-export/DataExport.js [hunk 4]
```diff
@@ -119,18 +121,53 @@ class DataExport extends Component {
...
+                    {!values.teamId && (
+                      <div className="d-flex flex-row">
+                        <InputPair
+                          view="switch"
+                          caption="Active employees"
+                          name="includeActiveEmployees"
+                          handleChange={checked => {
+                            setFieldValue(
+                              'includeActiveEmployees',
+                              checked ? true : false
```
Comment (aberonja) [inline src/main/frontend/src/pages/data-export/DataExport.js:133]: Same as below.

File: src/main/frontend/src/pages/data-export/DataExport.js (hunk 2 above)
Comment (slobodanc93) [reply src/main/frontend/src/pages/data-export/DataExport.js]: Yes, you're right, I've changed it to be just _checked_.

**Comment by stanislavgrujic** [comment]
@aberonja Could you please review?

File: src/main/frontend/src/pages/administration/components/common/DrawerElements.js [hunk 5]
```diff
@@ -252,11 +252,11 @@ const renderView = props => {
 };
 
 export const InputPair = props => {
-  const { separator, name, caption, view, disabled } = props;
+  const { separator, name, caption, view, disabled, leftPadding, flex } = props;
```
Comment (aberonja) [inline src/main/frontend/src/pages/administration/components/common/DrawerElements.js:255]: \n

File: .gitignore [hunk 6]
```diff
@@ -67,4 +67,5 @@ yarn-error.log*
 
 **.csv
 
-**/env
\ No newline at end of file
+**/env
+src/main/frontend/package-lock.json
```
Comment (aberonja) [inline .gitignore:71]: Why is package lock in the gitignore? Lock files should be available in the git.

**Comment by stanislavgrujic** [comment]
Please resolve merge conflicts.

**Pull request merged**